docs/dml-operation-on-carbondata.md

```diff
@@ -149,6 +149,50 @@ You can use the following options to load data:
    
    * If this option is set to TRUE, then high.cardinality.identify.enable property will be disabled during data load.
    
+- **SORT_SCOPE:** This property can have four possible values :
+
+    * BATCH_SORT : The sorting scope is smaller and more index tree will be created,thus loading is faster but query maybe slower.
+
+    * LOCAL_SORT : The sorting scope is bigger and one index tree per data node will be created, thus loading is slower but query is faster.
+
+    * GLOBAL_SORT : The sorting scope is bigger and one index tree per task will be created, thus loading is slower but query is faster.
+
+    * NO_SORT     : Feasible if we want to load our data in unsorted manner.
+
+    For BATCH_SORT:
+
+    ```
+    OPTIONS ('SORT_SCOPE'='BATCH_SORT')
+    ```
+
+    You can also specify the sort size option for sort scope.
+
+    ```
+    OPTIONS('SORT_SCOPE'='BATCH_SORT', 'batch_sort_size_inmb'='7')
+    ```
+
+    Note :
+
+    * batch_sort_size_inmb : Size of data in MB to be processed in batch. By default it is the 45 percent size of sort.inmemory.size.inmb(Memory size in MB available for in-memory sort).
+
+    For GLOBAL_SORT :
```

Comment:
Sorry for my mistake, now when sort_scope='GLOBAL_SORT', single_pass can be 'true',  I have raised a pr to remove this restriction of code (PR-1224).